src/lib/interactionManager/interaction.ts

```diff
@@ -49,8 +49,8 @@ export class Interaction {
   public channel: InteractionChannel
 
   public participants!: {
-    us: Identity
-    them: Identity
+    requester: Identity
+    responder?: Identity
```

Comment:
the SDK always knows it's own identity, and us/them wasnt descriptive of role

Reply:
the same applies to the SW. Maybe both should be optional?

Reply:
ah, but there is always a requester, you could not have an initial token without one

Reply:
true, nvm